### PR TITLE
Appveyor node bears

### LINKED ISSUE #2818: Node bears are not working on AppVeyor
All node bears are being skipped

### PR DESCRIPTION
**Pull request by jayvdb**
Closes #2818

### CODE REVIEW

File: .ci/appveyor.yml [hunk 1]
```diff
@@ -47,7 +47,21 @@ install:
     -r requirements.txt -r docs-requirements.txt"
   - ps: "Install-Product node ''"  # Use latest node v5.x.x
   - "npm config set loglevel warn"
+  - "sed -i '/alex/d' package.json"
+  - "sed -i '/coffeelint/d' package.json"
+  - "sed -i '/csscomb/d' package.json"
+  - "sed -i '/docker/d' package.json"
+  - "sed -i '/elm/d' package.json"
```
Comment (sangamcse) [inline .ci/appveyor.yml]: I am a bit confused about `alex` and `elm`. Other than these 2, all changed packages have been removed from `package.json`.

**Comment by jayvdb** [comment]
ack https://github.com/coala/coala-bears/pull/2819/commits/31e80bb79fe120ce579e41f800c02e373f9489a1 b7aa586

**Comment by jayvdb** [comment]
unack b7aa586

**Comment by jayvdb** [comment]
ack https://github.com/coala/coala-bears/commit/2888639aa2427ae77ab74486e8750770a1e219c1

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Pull request merged**